Dockerfile

```diff
@@ -1,41 +1,39 @@
-FROM ubuntu:22.04
-USER root
 ARG DEBIAN_FRONTEND=noninteractive
-
 # update and upgrade packages
 RUN apt-get -y update --fix-missing \
     && apt-get -y upgrade \
-    && apt-get -y update \
-    && apt-get -y autoremove
-
+    && apt-get -y install software-properties-common \
+    && add-apt-repository ppa:deadsnakes/ppa \
```

Comment:
I don't think these two are required.

Reply:
Ok, for me are required, but if it works without them, better to do not add them!